.github/workflows/run-setup-script.yml

```diff
@@ -0,0 +1,43 @@
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+on: [push, pull_request]
+name: Azure ARM
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+
+      # Checkout code
+    - uses: actions/checkout@main
+
+      # Log into Azure
+    - uses: azure/login@v1
+      with:
+        creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+    - name: Get git info
+      id: vars
+      shell: bash
+      run: |
+        echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+        echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
+
+    - name: Install Azure Infrastructure
+      run: ./deploy/setup.sh pr${{ steps.vars.outputs.sha_short }} eastus custom-vision-model-v2 pr${{ steps.vars.outputs.sha_short }}-environment stellarbatchdev
+      shell: bash
+
+    - name: Grant Admin role for synapse workspace to github service-principal and TM-Stellar AAD group
+      run: |
+        az synapse role assignment create --workspace-name pr${{ steps.vars.outputs.sha_short }}-pipeline-syn-ws --role "Synapse Administrator" --assignee 8c0d8761-c477-4ab0-b1b1-eb25d7355294
```

Comment:
Can you add comments here as to what the guids represent?

Reply:
fixed